uicomponent-compose/main/src/main/java/io/github/droidkaigi/feeder/AppContent.kt

```diff
@@ -80,13 +79,20 @@ fun AppContent(
                     }
                 )
             ) { backStackEntry ->
-                val feedType = backStackEntry
-                    .arguments?.getString("feedTab") ?: FeedTabs.Home.routePath
+                val routePath = rememberRoutePath(
+                    backStackEntry.arguments?.getString("feedTab")
+                        ?: FeedTabs.Home.routePath
+                )
+                val selectedTab = FeedTabs.ofRoutePath(routePath.value)
+                drawerContentState.onSelectDrawerContent(selectedTab)
                 val context = LocalContext.current
                 FeedScreen(
                     onNavigationIconClick = onNavigationIconClick,
-                    selectedTab = FeedTabs.ofRoutePath(feedType),
-                    onSelectedTab = { feedTabs -> actions.onSelectTab(feedTabs) },
+                    selectedTab = selectedTab,
+                    onSelectedTab = { feedTabs ->
+                        routePath.value = feedTabs.routePath
+                        drawerContentState.onSelectDrawerContent(feedTabs)
```

Comment:
Thank you! How about putting a comment like this? 🙏 

```suggestion
                        // We don't use navigation transitions here for animation.
                        routePath.value = feedTabs.routePath
                        drawerContentState.onSelectDrawerContent(feedTabs)
```

Reply:
Thanks for suggestion. I added a comment!